### ISSUE TITLE
release 2.2.0 was removed from ghcr.io

### ISSUE THREAD

**Issue by AndrewChubatiuk** [opencost/opencost#2683]
**Describe the bug**
several days ago tag v2.2.0 was pushed, which triggered an image build for that tag. Yesterday we've started using this image and today it disappeared from ghcr.io. Could you explain a reason or could you please restore it?

**Comment by cliffcolvin** [comment]
Hello @AndrewChubatiuk the 2.2 tag and package were an unintended release.

In December of 2023 we discussed in community meetings, and started down the path of  separating out some of the old Kubecost dependencies and named items in opencost. In doing that we not only moved opencost packages to ghcr.io/opencost but we also gave opencost its own build process. That legacy build process, though had one remaining piece that pushed the tag in question and was a defect after the separation.

We unfortunately didn't catch this issue until Monday after the KC release and removed it once it was identified. I have remedied the release script that caused this so the issue will not happen again in the future. I will also be discussing later today with Matt Ray when we are going to release 1.110.0 which will be the new release of opencost as planned.
